l10n_nl_state/README.rst

```diff
@@ -0,0 +1,85 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+============
+l10n_nl_state
+============
```

Comment:
use a human readable name here, and use the same in the `name` key of the manifest. And make this an rst title by have the same amount of equal signs as text